qt-bugs@ issue : none
Trolltech task ID : 171843
bugs.kde.org number : none
applied: yes
author: Laurent Montel <montel@kde.org>

This patch changed un-existing slot in qcombobox by existing slot.
(internalActivate is defined by q3combobox)
When qcombobox was transform to popupmenu when toolbar with combobox is too small
we used this slot to update combobox. (now it's updated)

--- qt4-x11-4.3.2.orig/src/qt3support/widgets/q3toolbar.cpp
+++ qt4-x11-4.3.2/src/qt3support/widgets/q3toolbar.cpp
@@ -667,7 +667,7 @@
                     Q3PopupMenu *cp = new Q3PopupMenu(d->extensionPopup);
                     cp->setEnabled(c->isEnabled());
                     d->extensionPopup->insertItem(s, cp);
-                    connect(cp, SIGNAL(activated(int)), c, SLOT(internalActivate(int)));
+                    connect(cp, SIGNAL(activated(int)), c, SLOT(setCurrentIndex(int)));
                     for (int i = 0; i < c->count(); ++i) {
                         QString tmp = c->text(i);
                         cp->insertItem(tmp, i);
@@ -682,7 +682,7 @@
                             Q3PopupMenu* sp = new Q3PopupMenu(d->extensionPopup);
                             cp->insertItem(tr("More..."), sp);
                             cp = sp;
-                            connect(cp, SIGNAL(activated(int)), c, SLOT(internalActivate(int)));
+                            connect(cp, SIGNAL(activated(int)), c, SLOT(setCurrentIndex(int)));
                         }
                     }
                 }
